picard/tagger.py

```diff
@@ -546,13 +546,17 @@ def handle_command_load(self, argstring):
         self.load_to_picard((argstring,))
 
     def handle_command_lookup(self, argstring):
-        argstring = argstring.upper()
-        if argstring == 'CLUSTERED':
-            self.autotag(self.clusters)
-        elif argstring == 'UNCLUSTERED':
-            self.autotag(self.unclustered_files.files)
+        if argstring:
+            argstring = argstring.upper()
         else:
+            argstring = 'ALL'
+        if argstring not in {'ALL', 'CLUSTERED', 'UNCLUSTERED'}:
             log.error("Invalid LOOKUP command argument: '%s'", argstring)
+            return
+        if argstring in {'ALL', 'CLUSTERED'}:
+            self.autotag(self.clusters)
+        if argstring in {'ALL', 'UNCLUSTERED'}:
+            self.autotag(self.unclustered_files.files)
```

Comment:
What about rewriting it as:

```python
    def handle_command_lookup(self, argstring):
        if argstring:
            argstring = argstring.upper()
        if not argstring or argstring == 'ALL':
            self.autotag(self.clusters)
            self.autotag(self.unclustered_files.files)
        elif argstring == 'CLUSTERED':
            self.autotag(self.clusters)
        elif argstring == 'UNCLUSTERED':
            self.autotag(self.unclustered_files.files)
        else:
            log.error("Invalid LOOKUP command argument: '%s'", argstring)
```

Reply:
That looks a bit cleaner.  I've pushed the changes.  Thanks.